src/Xamarin.Android.Build.Tasks/Tasks/ResolveSdksTask.cs

```diff
@@ -476,18 +476,16 @@ bool GetAapt2Version ()
 				);
 			} catch (Exception ex) {
 				Log.LogWarningFromException (ex);
-				Log.LogCodedWarning ("XA0034", $"Failed to get the Aapt2 version. Disabling Aapt2 Support.");
 				return false;
 			}
 			var versionInfo = sb.ToString ();
 			var versionNumberMatch = Aapt2VersionRegex.Match (versionInfo);
-			Version versionNumber;
-			if (versionNumberMatch.Success && Version.TryParse (versionNumberMatch.Groups ["version"]?.Value.Replace (":", "."), out versionNumber)) {
+			Log.LogMessage (MessageImportance.Normal, $"`{aapt2Tool} version` returned: ```{versionInfo}```");
```

Comment:
This should be `LogDebugMessage()`/low-priority message. *Most* of the time, nobody will want to know this value.